gradio/__init__.py

```diff
@@ -1,9 +1,32 @@
 import pkg_resources
 
-import gradio.components as components
-import gradio.inputs as inputs
-import gradio.outputs as outputs
 from gradio.blocks import Blocks, Column, Row, TabItem, Tabs
+from gradio.components import (
```

Comment:
@abidlabs are you sure about this? I think it is more proper to import components from `gr.components`. I am against importing all the components in the `init`. There are many components and a single component is not a critical element of gradio library, they need to referenced from `gr.components`

If people want to have shorter references they can import specific components via
`from gradio.components import Textbox, Json...`

Reply:
Hmm I really think the components and layouts both should be very easily imported since they are fundamental to building blocks. For me, `gr.Textbox` is a lot friendlier than `gr.components.Textbox`. 

There are basically 3 ways to make it friendly that I considered.

1. Doing `from gradio.components import Textbox, Image...` - the problem with this is that you have to explicitly import every component, which is annoying. Plus some of the components have common names (like `Image`), which could conflict with other classes in other libraries (like `from PIL import Image`)
2. Doing `from gradio.components import *` - bad practice, plus the same conflict issue
3. Making the components accessible as `gr.Textbox`, etc. I think this is way to go. The only potential problem is that if we have conflicts within the `gr.` namespace, which is unlikely.

Reply:
Well I still think the same, might hear thoughts from @aliabid94 and @pngwn if it will help.

Reply:
We need to be able to support `gr.Checkbox()` or `gr.Textbox()`. It is way too much extra typing to have to include gr.components every single time. This is how streamlit does it as well. There are tradeoffs to be made between user-friendliness and proper explicitness in building APIs and gradio definitely needs to lean towards user-friendliness.